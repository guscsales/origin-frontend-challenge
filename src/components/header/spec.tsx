import * as React from 'react';
import { create } from 'react-test-renderer';
import Header from '.';

describe('Header', () => {
  describe('#render', () => {
    it('should match a snapshot', () => {
      const wrapper = create(<Header />).toJSON();

      expect(wrapper).toMatchSnapshot();
    });
  });
});
