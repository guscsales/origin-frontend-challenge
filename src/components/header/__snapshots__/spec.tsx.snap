// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header #render should match a snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background: #fff;
  height: 5.6rem;
  padding: 0;
  margin-bottom: 3rem;
}

@media (min-width:1024px) {
  .c0 {
    -webkit-box-pack: start;
    -webkit-justify-content: flex-start;
    -ms-flex-pack: start;
    justify-content: flex-start;
    padding: 0 4rem;
    margin-bottom: 4.5rem;
  }
}

<div
  className="c0"
>
  <svg
    height="2.3rem"
    width="7rem"
  >
    <use />
  </svg>
</div>
`;
